sql/npc_list.sql

```diff
@@ -12714,9 +12714,9 @@ INSERT INTO `npc_list` VALUES (17212100,'Elrica','Elrica',0,0.000,0.000,0.000,0,
 INSERT INTO `npc_list` VALUES (17212101,'Terounalivet','Terounalivet',0,0.000,0.000,0.000,0,40,40,0,0,0,6,2075,0x0100060353105320533053405350006000700000,32,NULL,1);
 INSERT INTO `npc_list` VALUES (17212102,'qm1','???',0,309.600,2.600,324.000,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,0);
 INSERT INTO `npc_list` VALUES (17212103,'Cavernous_Maw','Cavernous Maw',0,462.628,0.029,479.280,32769,40,40,0,0,96,0,3,0x0000140900000000000000000000000000000000,2,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17212104,'apocalypse','     ',0,-217.283,97.267,461.827,32774,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,NULL,1);
-INSERT INTO `npc_list` VALUES (17212105,'no_name_0','     ',140,-258.186,-3.231,204.774,32769,40,40,0,0,0,0,27,0x0000160900000000000000000000000000000000,34,NULL,1);
-INSERT INTO `npc_list` VALUES (17212106,'no_name_1','     ',236,-276.303,12.310,261.650,32769,40,40,0,0,0,6,2075,0x0000160900000000000000000000000000000000,34,NULL,1);
+INSERT INTO `npc_list` VALUES (17212104,'no_name_0','     ',138,-281.182,40.113,263.102,32769,50,50,0,0,96,0,27,0x0000160900000000000000000000000000000000,34,NULL,1);
+INSERT INTO `npc_list` VALUES (17212105,'no_name_1','     ',140,-258.186,-3.231,204.774,32769,50,50,0,0,96,0,27,0x0000160900000000000000000000000000000000,34,NULL,1);
+INSERT INTO `npc_list` VALUES (17212106,'no_name_2','     ',236,-276.303,12.310,261.650,32768,50,50,0,0,96,6,2075,0x0000160900000000000000000000000000000000,34,NULL,1);
 INSERT INTO `npc_list` VALUES (17212107,'Lycopodium','Lycopodium',83,-275.953,12.333,262.368,32768,40,40,0,0,96,6,27,0x0000C70800000000000000000000000000000000,34,NULL,1);
```

Comment:
So I was going to say, "I don't know what's supposed to be at 17212104, but it probably shouldn't be moving and changing models."

_But_! [Looking at the capture you're using, it seems to be correct and does line up with the surrounding NPCs (qm1 through Field Manual)](https://github.com/KnowOne134/NPC-MOB-Logger/blob/master/Tables/North%20Gustaberg.lua#L475-LL482). So I'll vouch for ya here.

Reply:
Yes sorry I should have clarified that this was from the hardwork provided by @wrenffxi  and @UynGH  packet hunting 😸